### ISSUE TITLE
remeber export location

### ISSUE THREAD

**Issue by breadje** [lukos/simsig_importer#46]
Would it be possible to not only let the exporter remember the location of the imported excel, but also the location of the last exported WTT?

**Comment by lukos** [comment]
Yes, I need to work out how to do this. There is a built-in mechanism in .Net but it is not supported in the version of code I am using.

**Comment by lukos** [comment]
I think we could do this directly with the properties functionality, you just need to add some more properties to the ImporterProperties class and get/set them when dialogs are opened and closed. Need to use two settings, one for import and the other for export. You can use `Path.GetDirectory(path)` once the dialog has closed to extract the directory name from the file they have just chosen.
